pulse/common/src/main/java/gov/ca/emsa/pulse/common/domain/PatientRecordName.java

```diff
@@ -7,6 +7,7 @@
 	
 	private String id;
 	private ArrayList<GivenName> givens;
+	private ArrayList<String> givenStrings;
```

Comment:
Do you still need the "givens" array for something? I was expecting to see it changed to a list of Strings rather than have an additional property added.